package.json

```diff
@@ -13,7 +13,7 @@
     "start": "node lib",
     "dev": "node lib | ./node_modules/.bin/bunyan",
     "test": "npm run lint && CI=1 tap -R spec test/**/*.test.js --timeout 60",
-    "lint": "eslint -c .eslintrc cli lib test",
+    "lint": "eslint -c .eslintrc cli lib test; ./lintVerifier.sh",
```

Comment:
replace `;` with `&&`
Without it, `eslint` can fail but the process will succeed because the verifier is ok.